scalatest/src/main/scala/org/scalatest/Suite.scala

```diff
@@ -2153,7 +2153,7 @@ used for test events like succeeded/failed, etc.
     }
 
     val testNameArray =
-      for (m <- theSuite.getClass.getMethods; if isTestMethod(m))
+      for (m <- theSuite.getClass.getMethods.map(_.nn); if isTestMethod(m))
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.